core/src/wallet/pool/WalletPool.hpp

```diff
@@ -124,6 +124,8 @@ namespace ledger {
                 const std::shared_ptr<api::PreferencesBackend> &internalPreferencesBackend
             );
 
+
+
```

Comment:
We should avoid to introduce formatting changes since a formatting tool will be integrated soon